src/fragments/lib/auth/common/mfa/flows.mdx

```diff
@@ -0,0 +1,318 @@
+## SMS flows
+
+<Callout>
+Note: If you create or update an SMS MFA configuration for your Cognito user pool, the Cognito service will send a test SMS message to an internal number in order to verify your configuration. You will be charged for these test messages by Amazon SNS.
+
+For information about Amazon SNS pricing, see [Worldwide SMS Pricing](https://aws.amazon.com/sns/sms-pricing/).
+</Callout>
+
+
+There are a few ways to integrate phone numbers into an Amplify project's sign-in and verification process.
+- **As a username**\*: Users login with a username and password where their phone number acts as the username.
+- **As a verification method**: Users login by any means, but must verify their account with an OTP (one time password) sent to their phone.
+- **MFA (Multi-Factor Authentication)**: Users must verify every login with an OTP sent to their phone.
+
+\**Note: This is different from using a phone number [alias](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html#user-pool-settings-aliases), which is currently unsupported by the Amplify CLI.*
+
+These methods may be combined with each other or used independently but they all require the same prerequisites for sending SMS messages via Amazon SNS, the notification service used by Amplify.
+
+### Prerequisites
```

Comment:
for the sections on the right nav - we should have them be in the order:

- High level intro (doesn't need a header - and it should just be the name on the sidenav which is "Multi-factor Authentication".
- Set up backend resources (this section should detail out the experience for a developer using the CLI to add SMS or TOTP).
- Multi-factor authentication with SMS
- Multi-fcator authentication with TOTP

All the info here about the sign up and using phone numbers for registration should be removed. This guide should be all about MFA.

Reply:
Done.